scripts/simplesamlphp/acquia_config.php

```diff
@@ -38,172 +28,91 @@
  *
- * @return array
- *   Updated configuration.
- */
-function mc_session_store(array $config, array $ah_options) {
-  $config['store.type'] = 'memcache';
-  $config['memcache_store.servers'] = mc_info();
-  $config['memcache_store.prefix'] = $ah_options['database_name'];
-
-  return $config;
-}
-
-/**
- * Get memcache information.
+ * Warning: This has performance implications for anonymous users.
  *
- * @return array
- *   Memcache server pool information.
+ * @link https://docs.acquia.com/resource/using-simplesamlphp-acquia-cloud-site
  */
-function mc_info() {
-  $creds_json = file_get_contents('/var/www/site-php/' . getenv('AH_SITE_NAME') . '/creds.json');
-  $creds = json_decode($creds_json, TRUE);
-  $mc_server = [];
-  $mc_pool = [];
-  foreach ($creds['memcached_servers'] as $fqdn) {
-    $mc_server['hostname'] = preg_replace('/:.*?$/', '', $fqdn);
-    array_push($mc_pool, $mc_server);
-  }
-
-  return [$mc_pool];
-}
-
+// setcookie('NO_CACHE', '1');.
 /**
- * Set SQL database session storage.
+ * Generate Acquia session storage via hosting creds.json.
  *
- * @param array $config
- *   Current configuration.
- * @param string $database_name
- *   The name of a MySQL database.
+ * Session sorage defaults using the database for the current request.
  *
- * @return array
- *   Updated configuration.
+ * @link https://docs.acquia.com/resource/using-simplesamlphp-acquia-cloud-site/#storing-session-information-using-the-acquia-cloud-sql-database
  */
-function sql_session_store(array $config, $database_name) {
-  $creds = db_info($database_name);
+
+if (!getenv('AH_SITE_ENVIRONMENT')) {
+  // Add / modify your local configuration here.
   $config['store.type'] = 'sql';
-  $config['store.sql.dsn'] = sprintf('mysql:host=%s;port=%s;dbname=%s', $creds['host'], $creds['port'], $creds['name']);
-  $config['store.sql.username'] = $creds['user'];
-  $config['store.sql.password'] = $creds['pass'];
+  $config['store.sql.dsn'] = sprintf('mysql:host=%s;port=%s;dbname=%s', '127.0.0.1', '', 'drupal');
+  $config['store.sql.username'] = 'drupal';
+  $config['store.sql.password'] = 'drupal';
   $config['store.sql.prefix'] = 'simplesaml';
+  $config['certdir'] = "/var/www/simplesamlphp/cert/";
+  $config['metadatadir'] = "/var/www/simplesamlphp/metadata";
+  $config['baseurlpath'] = 'simplesaml/';
+  $config['loggingdir'] = '/var/www/simplesamlphp/log/';
 
-  return $config;
 }
-
-/**
- * Get SQL database information.
- *
- * @param string $db_name
- *   The name of a MySQL database.
- *
- * @return array
- *   Database information.
- */
-function db_info($db_name) {
-  $creds_json = file_get_contents('/var/www/site-php/' . getenv('AH_SITE_NAME') . '/creds.json');
+elseif (getenv('AH_SITE_ENVIRONMENT')) {
+  // Set  ACE ad ACSF sites based on hosting database and site name.
+  $config['certdir'] = "/mnt/www/html/{$_ENV['AH_SITE_GROUP']}.{$_ENV['AH_SITE_ENVIRONMENT']}/simplesamlphp/cert/";
```

Comment:
`/mnt/www/html/{$_ENV['AH_SITE_GROUP']}.{$_ENV['AH_SITE_ENVIRONMENT']}/simplesamlphp/` is pointing to the `simplesamlphp` symlink in the docroot from `symlinkDocrootToLibDir()` right? Using a relative path works and would make non-environment specific, i.e. `$config['certdir'] = 'cert/';`. Same for `$config['metadatadir']` on the next line

Reply:
@rjgwiz I believe the simplesamlphp_auth module has a note that it requires absolute paths for those files - have you found that not to be the case?

Reply:
@lcatlett yea - in our project we have been using relative path for over a year. Currently on [simplesamlphp](https://github.com/simplesamlphp/simplesamlphp) `v1.15.4`